clang/test/Sema/aarch64-streaming-sme-or-nonstreaming-sve-builtins.c

```diff
@@ -0,0 +1,51 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -target-feature +sve -fsyntax-only -verify %s
+
+// REQUIRES: aarch64-registered-target
+
+#include <arm_sve.h>
+
+__attribute__((target("+sve2p1")))
+svfloat32_t good1(svfloat32_t a, svfloat32_t b, svfloat32_t c) {
+  return svclamp(a, b, c);
+}
+
+__attribute__((target("+sme2")))
+svfloat32_t good2(svfloat32_t a, svfloat32_t b, svfloat32_t c) __arm_streaming {
+  return svclamp(a, b, c);
+}
+
+__attribute__((target("+sve2p1,+sme2")))
+svfloat32_t good3(svfloat32_t a, svfloat32_t b, svfloat32_t c) __arm_streaming_compatible {
+  return svclamp(a, b, c);
+}
```

Comment:
For completeness can we have a test as well to check this case but instead with `__arm_streaming` as well